6-conditions/ternary-operator.js

```diff
@@ -0,0 +1,50 @@
+/*
+-------------------------------------------------------------------------------------
+ Tutorial: Ternary Operator 
+-------------------------------------------------------------------------------------
+*/
+/* Code here explaining concept with comments to guide */
+// The Ternary operator is way of make a conditional decisions
+// Is a simple way of make an if-else block 
+
+// if-else block example I
+let age = 17;
+
+if( age > 18 ){
+    console.log("You are an adult");
+} else{
+    console.log("You are not an adult");
+}
+
+// ternary-operator example II
+age = 42;
+let x = age > 18 ?
+        console.log("You are an adult") :
+        console.log("You are not an adult");
+
+console.log(x);
+
+/*** Ternary Operator Structure
+    condition ?
+    "The condition is True" :
+    "The condition is false"
+***/
+
+// ternary-operator example III ( return value)
+age = 10;
+let msg = age > 18 ?
+          "You are an adult" :
+          "You are not an adult";
+
+console.log(msg);
```

Comment:
Example 2 and 3 are technically both return value, I think you can keep only one exemple.

You should also put "you are not a adult" or "you are an adult" messages in a variable, it will be more easy to explain in comment the difference between ternary operator and if-else block and see the benefits to use it.

Reply:
I get it.